cli_test/test_helpers.go

```diff
@@ -757,13 +758,15 @@ func WriteToNewTempFile(t *testing.T, s string) *os.File {
 	return fp
 }
 
```

Comment:
If these two below are unused, why not remove them?

Reply:
they are being used, its a false positive since they are unexported but used in the pkg